test/cookbooks/macos_test/recipes/new_users.rb

```diff
@@ -7,5 +7,6 @@
 
 macos_user 'create non-admin user johnny' do
   username 'johnny'
+  fullname 'Johnny\ Appleseed'
```

Comment:
Should be fixable with [`Shellwords#shellescape`](https://ruby-doc.org/stdlib-2.3.0/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape):

```ruby
argv = "It's better to give than to receive".shellescape
argv #=> "It\\'s\\ better\\ to\\ give\\ than\\ to\\ receive"
```

Reply:
If this is being passed to the `command` method in an `execute` resource, we can pass `command` an array of strings and it will automatically escape them for us. We shouldn't need to manually escape any strings ourselves; let the tools do that job for us.